DFC.App.Account.Services.SHC/Models/ShcDocumentResponse.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System.Collections.Generic;
+using System.Xml.Serialization;
+
+namespace DFC.App.Account.Services.SHC.Models
+{
+	[XmlRoot(ElementName = "UpdatedAt", Namespace = "http://schemas.skillsfundingagency.bis.gov.uk/alse/skillscentral/1.0/Entities")]
+	public class UpdatedAt
+	{
+		[XmlAttribute(AttributeName = "nil", Namespace = "http://www.w3.org/2001/XMLSchema-instance")]
+		public string Nil { get; set; }
+	}
+
+	[XmlRoot(ElementName = "DeletedAt", Namespace = "http://schemas.skillsfundingagency.bis.gov.uk/alse/skillscentral/1.0/Entities")]
+	public class DeletedAt
+	{
+		[XmlAttribute(AttributeName = "nil", Namespace = "http://www.w3.org/2001/XMLSchema-instance")]
+		public string Nil { get; set; }
+	}
+
+	[XmlRoot(ElementName = "DataValues", Namespace = "http://schemas.skillsfundingagency.bis.gov.uk/alse/skillscentral/1.0/Entities")]
+	public class DataValues
+	{
+		[XmlAttribute(AttributeName = "nil", Namespace = "http://www.w3.org/2001/XMLSchema-instance")]
+		public string Nil { get; set; }
+	}
+
+	[XmlRoot(ElementName = "SkillsDocumentIdentifier", Namespace = "http://schemas.skillsfundingagency.bis.gov.uk/alse/skillscentral/1.0/Entities")]
+	public class SkillsDocumentIdentifierResponse
+	{
+		[XmlElement(ElementName = "ServiceName", Namespace = "http://schemas.skillsfundingagency.bis.gov.uk/alse/skillscentral/1.0/Entities")]
```

Comment:
Is it possible to have the Namespace as a constant?

Reply:
Updated.